README.md

```diff
@@ -97,22 +97,23 @@ Active namespace is "default".
 
 There are several installation options:
 
-- As kubectl plugins (macOS/Linux)
-- macOS
-  - Homebrew (recommended)
-  - MacPorts
-- Linux
-  - Debian
-  - Arch Linux
-  - Homebrew
-  - Manual installation
+- [as kubectl plugins (macOS & Linux)](#kubectl-plugins-macos-and-linux)
+- [with Homebrew (macOS & Linux)](#homebrew-macos-and-linux)
+- [with MacPorts (macOS)](#macports-macos)
+- [with apt (Debian)](#apt-debian)
+- [with pacman (Arch Linux)](#pacman-arch-linux)
+- [manually (macOS & Linux)](#manual-installation-macos-and-linux)
+  
+If you like to add context/namespace information to your shell prompt (`$PS1`), you can try out [kube-ps1](https://github.com/jonmosco/kube-ps1).
+  
 
 ### Kubectl Plugins (macOS and Linux)
 
-You can install and use [Krew](https://github.com/kubernetes-sigs/krew/) kubectl
-plugin manager to get `kubectx` and `kubens`. **NOTE:** This will not install
-shell completion scripts, if you want those, choose another installation method
-below.
+You can install and use the [Krew](https://github.com/kubernetes-sigs/krew/) kubectl
+plugin manager to get `kubectx` and `kubens`.
+
+ **NOTE:** This will not install the shell completion scripts. If you want them, choose another installation method
```

Comment:
There's an extra space before the format markers `**Note**`

Reply:
Done.